lib/metrics/drivers/statsd.rb

```diff
@@ -0,0 +1,46 @@
+module Metrics::Drivers
+  class Statsd
+    attr_reader :client, :template, :source_prefix
+
+    def initialize(client, template, source_prefix)
+      @client        = client        # The statsd client
+      @template      = template      # The template for the metric name
+      @source_prefix = source_prefix # Source prefix
+    end
+
+    def write(*instrumenters)
+      instrumenters.each do |instrumenter|
+        emit(instrumenter)
+      end
+      instrumenters
+    end
+
+    def emit(instrumenter)
```

Comment:
Minor and totally my fault, but I kinda dislike the word `instrumenter`. I think this would be easier to read if we just had a `Metric` type, like we do in the Go version of this.

Maybe in a v2 version of this lib.